views/place-form.hbs

```diff
@@ -1,211 +1,219 @@
-    <div>
-      <label for="restaurant">{{localText.restaurant}}</label>
-      <input id="restaurant" type="checkbox" name="categories" value="food and drink" {{checkOptionsValue place.categories "restaurant"}}>
-    </div>
-    <div>
-      <label for="education">{{localText.education}}</label>
-      <input id="education" type="checkbox" name="categories" value="education" {{checkOptionsValue place.categories "education"}}>
-    </div>
-    <div>
-      <label for="retail">{{localText.retail}}</label>
-      <input id="retail" type="checkbox" name="categories" value="retail" {{checkOptionsValue place.categories "retail"}}>
-    </div>
-    <div>
-      <label for="sport">{{localText.sport}}</label>
-      <input id="sport" type="checkbox" name="categories" value="sport" {{checkOptionsValue place.categories "sport"}}>
-    </div>
-    <div>
-      <label for="municipal">{{localText.municipal}}</label>
-      <input id="municipal" type="checkbox" name="categories" value="municipal" {{checkOptionsValue place.categories "municipal"}}>
-    </div>
-    <div>
-      <label for="healthcare">{{localText.healthcare}}</label>
-      <input id="healthcare" type="checkbox" name="categories" value="healthcare" {{checkOptionsValue place.categories "healthcare"}}>
-    </div>
-  </fieldset>
-
-  <input type="submit" value="{{localText.formSubmit}}">
+{{>header}}
+{{>options}}
+
+<div class="tc w-100 pt5 mt2">
+  <span class="dark-text">{{localText.formInfo}}</span>
+</div>
+{{#if edit}}
+  <form class="" action="/{{lang}}/edit-place/{{place._id}}" method="post">
+    <input type="hidden" name="_method" value="put">
+{{else}}
+  <form class="" action="/{{lang}}/add-place" method="post">
+    <input type="hidden" name="_method" value="post">
+{{/if}}
+<table class="f6 w-100 mw8 center bb b--turquoise bw2">
+  <tbody>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">*{{localText.name}}</td>
+      <td class="w-100">
+          {{#if place.en.name}}
+            <input class="input-reset w-100 br2 ba b--gray" type="text" name="name_en" value="{{place.en.name}}">
+          {{else}}
+            <input class="input-reset w-100 br2 ba b--gray dib" type="text" name="name_en" placeholder="{{localText.english}}">
+          {{/if}}
+      </td>
+    </tr>
+    <tr>
+        <td class="w-20"></td>
+        <td class="w-100">
+          {{#if place.ar.name}}
+            <input class="input-reset w-100 br2 ba b--gray" type="text" name="name_ar" value="{{place.ar.name}}">
+          {{else}}
+            <input class="input-reset w-100 br2 ba b--gray dib" type="text" name="name_ar" placeholder="{{localText.arabic}}">
+          {{/if}}
+        </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">{{localText.description}}</td>
+      <td class="w-100 pt3">
+          {{#if place.en.description}}
+            <textarea class="input-reset w-100 br2 ba b--gray" type="text" name="description_en" value="{{place.en.description}}"></textarea>
```

Comment:
Because description is now a text area `value` isn't a thing anymore, in needs to go between the two tags:
```html
<textarea class="input-reset w-100 br2 ba b--gray" type="text" name="description_en">
{{place.en.description}}</textarea>
```

Reply:
make sure to do this on all the different text areas